meetings/2024-03-06.md

```diff
@@ -45,12 +50,93 @@ URL: https://github.com/solid/specification/discussions/618
+* HZ: Was at Data Transfer Initiative. Talking about portability. Aware of Solid but wondering why work out there at W3C. IMO
+* PAC: Thanks all for sharing thoughts. Others have said before re branding, and liability. I like what eP said what the Group will do. Perhaps that's also how we present. To say that initiatives such as Solid. Which is differen tthan we are goin gto make Solid a rec. I expect than for Solid to be a thing. Then telling W3C Members we are not 
+* SC: Social Web had understanding about what Social meant. Controlling own data / Personal / Protected also brings some meaning, several initiatives touch area of people controling their own data. This is not a recent topic. If the group is focusing on personal compared to social, ..., the specs we come up with here are just scratching the surface when it comes to fixing the web. We need other gruops to also bring their expetise.
+* eP: Good to stay close to Linked Data Platform. Not sure if we should make it more prominent. At least point it to the paragraph in the Solid Protocol spec. What's missing or helpful towards LDP.
+* RG: Minor nit: Difference on protocols. Solid is working with any data not just personal. I think that needs to be emphasised as well. Not just contacts etc.
+* TBL: I think it is important that it is not just private data. A lot of it public, government data, maps. We need agreement in this group. These are not global. Some maybe restricted to the group. Most of the values on the Web is not just personal or completely private or public but in between. From the point of data spectrum, that's something Solid needs to address.
+
+
 ### Demo
 
 ### Special Topic Meetings
 URL: https://github.com/orgs/solid/projects/16/views/1
-* Solid Notifications discussed on 03/05 
+* HZ: Solid Notifications discussed on 2024-03-05. If you have proposals for upcoming STMs we can add to schedule.
+* RG: Can we have a slot for Solid Notifications?
+* HZ: Yes.
+* TBL: It has been awhile since we had an STM.
+* HZ: We had them in last weeks
+* SC: Was Solid Notifications discussed yesterday?
+* RG: Only audio.
+* eP: I guess we're not updating the CG calendar? Perhaps synchronise the board.
+* HZ: The meetings are in the Solid Calendar
+* TBL: There is a Discussion 555 which is where last discussions were mentioned. https://github.com/solid/specification/discussions/555
+* SC: We moved it to a project board, the discussion is now closed and chairs are maintaining the board. As far as the calendar, CG and STM have fixed slots. I used to update specific meetings to point to hackmd and the agenda. Right now the agenda is only mentioned in matrix chat, we could improve how we get the details out. https://github.com/orgs/solid/projects/16
```

Comment:
Note that #555 still *looks* alive. Someone coming back to it, to see the latest addition to the bottom of the table, won't necessarily see the top-posted notice that it's been closed with new info being posted at . I suggest adding `<strike>` or similar to make clear what is no longer valid. https://github.com/orgs/solid/projects/16 requires more thinking, parsing the date column which is no longer in ISO-numerics, figuring out that "status — done" means the meeting has been held... and now, it's even more challenging, because the last dated entry is March 5 (which is now in the past) and shows status "discussed" instead of "done" or at least "scheduled" which *seems* to make more sense for a date in the past....
```suggestion
* TBL: There is a [Discussion 555](https://github.com/solid/specification/discussions/555) which is where last discussions were mentioned. 
* SC: We moved it to [a project board](https://github.com/orgs/solid/projects/16). The discussion is now closed and chairs are maintaining the board. As far as the calendar, CG and STM have fixed slots. I used to update specific meetings to point to hackmd and the agenda. Right now the agenda is only mentioned in matrix chat; we could improve how we get the details out. 
```